.github/workflows/build-windows-executable-app.yaml

```
@@ -1,410 +1,427 @@
-  PYTHON_VERSION: 3.11.0
-  # Name of the installer
-  APP_NAME: OpenMS-StreamlitTemplateApp
-  # Define unique GUID for UpgradeCode
-  APP_UpgradeCode: "8d28e8c7-45dc-446c-b889-99a6aea2f1a5"
-  # Define needed TOPP tools here
-  TOPP_TOOLS: "FeatureFinderMetabo FeatureLinkerUnlabeledKD SiriusExport"
+   OPENMS_VERSION: 3.2.0
+   PYTHON_VERSION: 3.11.0
+   # Name of the installer
+   APP_NAME: OpenMS-StreamlitTemplateApp
+<<<<<<< Updated upstream
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 14: could not find expected ':'

(syntax-check)